### ISSUE TITLE
Electrum 2.8.3 fails to open wallet from 2.7.15, error parsing contacts

### ISSUE THREAD

**Issue by KeefeJ** [spesmilo/electrum#2523]
After upgrading from 2.7.15 to 2.8.3, I am no longer able to open my wallet. I get the error below.
```
$ electrum daemon &
$ electrum -w .electrum/wallets/problem_wallet daemon load_wallet
Traceback (most recent call last):
  File "/usr/local/bin/electrum", line 412, in <module>
    result = server.daemon(config_options)
  File "build/bdist.linux-x86_64/egg/jsonrpclib/jsonrpc.py", line 288, in __call__
  File "build/bdist.linux-x86_64/egg/jsonrpclib/jsonrpc.py", line 238, in _request
  File "build/bdist.linux-x86_64/egg/jsonrpclib/jsonrpc.py", line 567, in check_for_errors
jsonrpclib.jsonrpc.ProtocolError: (-32603, u'Server error:   File "/usr/local/lib/python2.7/dist-packages/electrum/contacts.py", line 41, in __init__ | ValueError: dictionary update sequence element #0 has length 34; 2 is required')
```

For privacy and security reasons, I won't provide the entire wallet file here, but I can provide redacted pieces as necessary for debugging. Below is the section of the wallet that I suspect is relevant. Let me know if more info is needed.
```
    "contacts": [
        "1B********************************",
        "1H********************************",
        "1C********************************",
        "1B********************************",
        "1B********************************",
        "1E*******************************",
        "1D********************************"
    ],
```

After deleting that section from the wallet, electrum opens it successfully.

**Comment by KeefeJ** [comment]
I just noticed that this is not a new issue. Although previous reports were closed, it appears the underlying bug still remains. Please consider fixing the bug, so users don't have to manually edit their wallet file after upgrading electrum. This bug is particularly troublesome for windows users who are unable to see the error message so have no clue why the app crashes immediately.

**Comment by KeefeJ** [comment]
Apparently this issue was fixed months ago, but the fix wasn't included in the 2.8.3 release a couple weeks ago?
https://github.com/spesmilo/electrum/commit/f46b403a8d2ffc382d12b273932bd73d7127df03
